src/components/ProjectList.scss

```diff
@@ -133,3 +175,37 @@
     }
   }
 }
+
+@media (min-width: 1600px) {
+  .projects-list-wrapper {
+    .project-item {
+      height: 450px;
+      // display: flex;
+      // justify-content: space-between;
+      .project-img-wrapper {
+        overflow: hidden;
+        width: 52%;
+        height: 100%;
+        flex-shrink: 0;
+        .project-img {
+          // position: absolute;
+          top: 0;
+          left: 0;
+          min-width: 550px;
+          height: 100%;
+        }
+      }
+      .right {
+        order: 1;
+      }
+      .project-text {
+        .project-title {
+          font-size: 3.5rem;
+        }
+        .project-description {
+          font-size: 2rem;
+        }
+      }
+    }
+  }
+}
```

Comment:
Added for large screen viewing to adjust font sizes and project sizing so pictures look scaled properly